### ISSUE TITLE
Unexpectedly found nil while implicitly unwrapping an Optional value

### ISSUE THREAD

**Issue by nathangabriel27** [react-native-text-input-mask/react-native-text-input-mask#306]
Unexpectedly found nil while implicitly unwrapping an Optional value

![image](https://github.com/user-attachments/assets/a13c4634-31e6-427f-bcef-17279cac2fde)

    "react-native": "0.76.4",
     "react-native-text-input-mask": "^3.2.0",

**Comment by YBeletsky117** [comment]
+1
I confirm that I also encounter this error and cannot build the application.
React Native version 0.76.4

**Comment by YBeletsky117** [comment]
@nathangabriel27 
Try it this way, it helped me
```
    var methodQueue: DispatchQueue {
        return bridge?.uiManager?.methodQueue ?? DispatchQueue.main
    }
```

**Comment by huziidroid** [comment]
> @nathangabriel27 Try it this way, it helped me
> 
> ```
>     var methodQueue: DispatchQueue {
>         return bridge?.uiManager?.methodQueue ?? DispatchQueue.main
>     }
> ```

This fixes the error, but results in unexpected behaviour of application. Sometimes mask is applied and sometimes it doesn't

**Comment by StephenTANM** [comment]
I can confirm I am also getting this error

**Comment by StephenTANM** [comment]
Looks like it might be due to the new architecture changes https://reactnative.dev/blog/2024/10/23/the-new-architecture-is-here#new-native-modules

**Comment by jamesmcn1** [comment]
Yeah i'm also getting this error. I can confirm this fix allows me to build the application:

```
    var methodQueue: DispatchQueue {
        return bridge?.uiManager?.methodQueue ?? DispatchQueue.main
    }
```

Seems stable enough but i haven't tested the fields thoroughly.

Not sure this library is maintained anymore.

**Comment by IvanIhnatsiuk** [comment]
@jamesmcn1 you can use [this](https://github.com/IvanIhnatsiuk/react-native-advanced-input-mask) one instead 🙂

**Comment by yaroslav-beletsky** [comment]
@IvanIhnatsiuk 
Hi! I looked at your package, it looks pretty cool! You've even introduced support for a new architecture, which is great!
Thanks! We'll keep trying
